contracts/ReputationMiningCycleRespond.sol

```diff
@@ -546,4 +498,14 @@ contract ReputationMiningCycleRespond is ReputationMiningCycleStorage, PatriciaT
     // Save the index for tiebreak scenarios later.
     disputeRounds[u[U_ROUND]][u[U_IDX]].provedPreviousReputationUID = previousReputationUID;
   }
+
+  function concatenateToBytes64(uint256 a, uint256 b) internal pure returns (bytes) {
+    bytes memory retValue = new bytes(64);
+    assembly {
+      // Seems as if you access an external variable by name, you get the value, straight up...?
```

Comment:
What do you mean "you get the value" I think you get the variable location in memory slot